pagopa/io-app
# feat: [IOCOM-1905] Handling of InApp Browser opening failure for FIMS SSO

## Short description
This PR adds the handling of the InApp Browser opening failure for the FIMS Authentication flow.

|iOS|Android|
|-|-|
|![iOS](https://github.com/user-attachments/assets/8da74aaf-0c5e-4ab8-b2f9-32a4a3dafb20)|![Android](https://github.com/user-attachments/assets/28d2e473-fc5d-4fea-8fc3-007e83b0e544)|

## List of changes proposed in this pull request
- InApp Browser opening call has been wrapped in a try catch in order to catch the error, track it and warn the user with a Toast.

## How to test
Using the io-dev-api-server, edit the native code to reject the promise (following a standard error generation) and check that the error is properly tracked and displayed.
